### PR TITLE
Add matchLevel to KycDocument

### PR DESCRIPTION
**Pull request by subenson**

### CODE REVIEW

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 1]
```diff
@@ -37,6 +37,9 @@ properties:
       - passport
       - id-card
       - driver-license
+  matchLevel:
+    description: The level of strictness for the document matches.
+    type: string
```
Comment (XaosSintez) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:42]: Is it string? Can you please add an example?

Comment (svensono) [reply openapi/components/schemas/Common/CommonKycDocument.yaml]: Oh, no it should be integer. I updated it.

**Review by XaosSintez — DISMISSED** [review]

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 2]
```diff
@@ -37,6 +37,10 @@ properties:
       - passport
       - id-card
       - driver-license
+  matchLevel:
+    description: The level of strictness for the document matches.
```
Comment (slavcodev) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:41]: Does it have any constraints? minimum, maximum or may be predefined levels?

Comment (svensono) [reply openapi/components/schemas/Common/CommonKycDocument.yaml:41]: Yes. I added them.

**Review by colinbird — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**